client/www/pages/docs/init.md

```diff
@@ -78,3 +54,33 @@ const db = init({ appId: APP_ID, schema });
 ```
 
 To learn more about writing schemas, head on over to the [Modeling your data](/docs/modeling-data) section.
+
+## Flexible Initialization
+
+Instant maintains a single connection regardless of where or how many times you
+call `init` with the same app ID. This means you can safely call `init` multiple
+times without worrying about creating multiple connections or
+performance overhead. However we do recommend the pattern of exporting a
+reference from a utility file like so:
+
+```typescript
+// lib/db.ts
+import { init } from '@instantdb/react';
+import schema from '../instant.schema';
+
+// Instant app
+const APP_ID = '__APP_ID__';
+export const db = init({ appId: APP_ID, schema });
+
+// app/page.tsx
+"use client";
+
+import { db } from '../lib/db';
+
+function App() {
+  // do some instant magic 🪄
+  db.useQuery({ ... });
```

Comment:
Thought: perhaps we should fill this in with `{ posts: { } } `

Or something like that?